src/PopOver.tsx

```diff
@@ -196,25 +196,36 @@ export interface PopOverProps {
     hotspotPath: string[],
     update: HotspotUpdate | null,
   ) => void;
-  changeScene: (id: string) => void;
+  photosphereOptions?: string[];
+  currentPS: string;
+  changeScene?: (id: string) => void;
```

Comment:
This field isn't supposed to be optional. It should be `changeScene: (id: string) => void;`